source/standards/monitoring.html.md.erb

```diff
@@ -1,13 +1,13 @@
 ---
 title: How to monitor your service
-last_reviewed_on: 2020-10-23
+last_reviewed_on: 2021-04-25
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
 
-At GDS, we follow the Service Manual guidance on how to [monitor the status of services][] and set [performance metrics][].
+In the CDIO and GDS, we follow the Service Manual guidance on how to [monitor the status of services][] and set [performance metrics][].
```

Comment:
Should we specify Cabinet Office CDIO as there are multiple CDIOs across government?

```suggestion
In the Cabinet Office CDIO and GDS, we follow the Service Manual guidance on how to [monitor the status of services][] and set [performance metrics][].
```